addons/website/views/website_templates.xml

```diff
@@ -520,6 +546,18 @@
     </xpath>
 </template>
 
+<!-- slideout effect -->
+<template id="template_footer_slideout" inherit_id="website.layout" name="Template Footer Slideout" active="False">
+    <xpath expr="//footer[@id='bottom']" position="attributes">
+        <attribute name="t-attf-class" add="o_footer_slideout" separator=" "/>
+
```

Comment:
Missing two closings tags, I will let you review your code and retest before checking